userguide/config.toml

```diff
@@ -3,8 +3,26 @@ title = "Docsy Example"
 
 enableRobotsTXT = true
 
-# Hugo allows theme composition (and inheritance). The precedence is from left to right.
+# set docsy as theme, either locally or as module
 theme = ["docsy"]
+# theme = ["github.com/google/docsy", "github.com/google/docsy/module"]
+
+# when using docsy as module, instead of using
+# theme = ["github.com/google/docsy", "github.com/google/docsy/module"]
+# you may alternatively use the declaration given below:
+#[module]
+#  # uncomment lines below for temporary local development of module
+#  # replacements = "github.com/google/docsy -> ../../docsy"
+#  # replacements = "github.com/google/docsy/module -> ../../docsy"
+#  [module.hugoVersion]
+#    extended = true
+#    min = "0.73.0"
+#  [[module.imports]]
+#  # pull in docsy as hugo module
+#    path = "github.com/google/docsy"
+#  [[module.imports]]
+#  # pull in workaround and dependencies
+#    path = "github.com/google/docsy/module"
```

Comment:
I'd rather that there be no changes to the `userguide` directory.

The lines above should find their way into doc pages, if anywhere IMHO.

Reply:
Lines 10-25 were removed in e444964a013e.